CHANGELOG.md

```diff
@@ -1,3 +1,66 @@
+Version 1.17.0
+==============
+**July 24, 2023**
+
+This release brings one of the most requested features - interactive twin-axis support! Another feature added in this release is the ability to easily set custom options on plot components with `backend_opts`, making it much easier to customize your plots. Datashaders `where` and `summary` aggregators are now supported, and `rasterize` now has a `selector` option, making it easy to get extra information about your rasterized data. Lastly, Bokeh figures with the same labels will synchronize muteness or visibility across different figures.
+
+In addition, this release includes several enhancements and bug fixes.
+
+Many thank to the new contributors @alfredocarella and @RaulPL, as well as the returning contributors @ahuang11, @droumis, @jbednar, @jlstevens, @maximlt, @philippjfr, @TheoMathurin and @Hoxbro.
+
+New features:
+
+- Multi-yaxis support in the Bokeh backend ([#5621](https://github.com/holoviz/holoviews/pull/5621))
+- Support custom options on plot components for the different backends ([#4463](https://github.com/holoviz/holoviews/pull/4463))
```

Comment:
I believe this could be
"Access and customize the underlying Bokeh or Matplotlib figure through `backend_opts`, an alternative, more straightforward version of `hooks`."